src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/util/Writer.java

```diff
@@ -646,4 +607,13 @@ public void do304(
         });
     }
 
+    public void send500(
+        MessageConsumer receiver,
+        long routeId,
+        long streamId,
+        long traceId)
+    {
+        doHttpResponse(receiver, routeId, streamId, traceId, e ->
+            e.item(h -> h.name(STATUS).value("500")));
+    }
```

Comment:
Does this method still need to be added now that we are not treating missing `httpBeginEx` as a special case?

Reply:
remove it. Not using it anymore.